client/pkg/transport/keepalive_listener_openbsd.go

```diff
@@ -0,0 +1,12 @@
+//go:build openbsd
```

Comment:
Please add license header something like below (change 2015 to 2023),

https://github.com/etcd-io/etcd/blob/5021cd924cb642636fed6b4cb499e2001f15a1c4/server/main.go#L1-L21